modules/build/src/main/scala/scala/build/Build.scala

```diff
@@ -825,9 +825,19 @@ object Build {
             Seq(ScalacOpt("-scalajs"))
           else Nil
 
-        val scalacReleaseV = releaseFlagVersion
-          .map(v => List("-release", v).map(ScalacOpt(_)))
-          .getOrElse(Nil)
+        val scalacReleaseV = {
+          val maybeScalaPatchVersion =
```

Comment:
IMO we should move this function for extracting patch version to `ScalaParameters`, it can be useful in another place.

Reply:
`ScalaParameters` is a part of [coursier/dependency](https://github.com/coursier/dependency)
I extracted the logic to `scala.build.options.ScalaVersionUtil` instead.